### ISSUE TITLE
Apply standardization as a post-processing option

### ISSUE THREAD

**Issue by tk3369** [khkwong/Normalize.jl#16]
Provide an option at the top-level transform function such that transformed data (from BoxCox, YeoJohnson, etc.) is further standardized using the center/scale algorithm from #11.

I would make that the default setting, which is the behavior from Scikit-Learn as well https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.PowerTransformer.html#sklearn.preprocessing.PowerTransformer
